### PR TITLE
Support standard color for app buttons block

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Pass color option to app buttons to turn them black on white (standard) background.
Add type guard function to keep things type safe. Maybe a bit complicated solution but at least it's type safe - what do you say?

## Why?

App buttons are white on white (standard) background. We are not passing the color option to the buttons component.

## Screenshot: hover vs. not-hover

Screenshot 2021-05-20 at 15 02 26

### CODE REVIEW

File: src/blocks/AppButtonsBlock.tsx [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+const STANDARD_COLORS: Array<minimalColorComponentColors> = [
+  'standard',
+  'standard-inverse',
+]
+
+function isStandardColor(
+  color: minimalColorComponentColors,
+): color is StandardColor {
+  return STANDARD_COLORS.includes(color)
+}
```
Comment (gustaveen) [inline src/blocks/AppButtonsBlock.tsx:20]: Might be just me that's too used to arrow functions :D But we try to stick to that for consistency or WDYT?

**Review by gustaveen — COMMENTED** [review]
Cool! I'm good with this solution but since you still can choose the options in Storyblok this might be confusing :)

File: src/blocks/AppButtonsBlock.tsx (hunk 1 above)
Comment (robinandeer) [reply src/blocks/AppButtonsBlock.tsx:20]: Sure, I don't mind either way! #tabsVSspaces

**Comment by robinandeer** [comment]
> Cool! I'm good with this solution but since you still can choose the options in Storyblok this might be confusing :)

I agree, but I guess we don't even intend to support all colors, no? Happy to take any suggestions!

**Review by gustaveen — APPROVED** [review]
🙌

**Pull request merged**